### PR TITLE
Make things a bit easier ot understand/absorb

### PR DESCRIPTION
**Pull request by NoComment1105**
Just changes a few word orders/choices here and there to make things clearer to the end reader.

### CODE REVIEW

File: docs/mixin-injects.md [hunk 1]
```diff
@@ -22,13 +22,13 @@ This tells the compiler that this is a mixin class, targeting the class `net.min
...
 
-`@Inject` is a mixin annotation. It tells the mixin compiler to "inject" the logic in our method to the part of the method `onBreak` specified by the `at` argument. `@At("HEAD")` refers to the top of the method target; the first bit of code to be run when that method is called.
+`@Inject` is a mixin annotation. It tells the mixin compiler to "inject" the logic of our method into the part of the `onBreak` method we specified with the `at` argument. `@At("HEAD")` refers to the top of the method target; the first bit of code to be run when that method is called.
 
 We then have a method with the same arguments as our target method, plus a `CallbackInfo` object.
 
 #### Registering Mixins
 
-When you add a mixin, you must put the mixin class in `your_package.mixin`. Then, you should add an entry to the `server` array in your `<your_modid>.mixins.json` file with the name of your mixin class.
+When you add a mixin, you must put the mixin class in `<your_package>.mixin`. Then, you should add an entry to the `server` array in your `<your_modid>.mixins.json` file with the name of your mixin class.
```
Comment (Tom-The-Geek) [inline docs/mixin-injects.md]: This line should probably also be adjusted to say that the `server` array is specifically for mixins that ONLY load on the dedicated server, and most things should just be placed in `mixins`

File: docs/mixin-injects.md [hunk 2]
```diff
@@ -22,13 +22,13 @@ This tells the compiler that this is a mixin class, targeting the class `net.min
 
 `@Inject(at = @At("HEAD"), method = "onBreak")`
 
-`@Inject` is a mixin annotation. It tells the mixin compiler to "inject" the logic in our method to the part of the method `onBreak` specified by the `at` argument. `@At("HEAD")` refers to the top of the method target; the first bit of code to be run when that method is called.
+`@Inject` is a mixin annotation. It tells the mixin compiler to "inject" the logic of our method into the part of the `onBreak` method we specified with the `at` argument. `@At("HEAD")` refers to the top of the method target; the first bit of code to be run when that method is called.
```
Comment (Tom-The-Geek) [inline docs/mixin-injects.md]:
```suggestion
`@Inject` is a Mixin annotation. It tells the Mixin runtime to "inject" the logic of our method into the `onBreak` method we specified with the `method` argument. The `at = @At("HEAD")` part means that our code will be injected at the top of the method; making it the first thing to run run when that method is called.
```

File: docs/setup.md [hunk 3]
```diff
@@ -56,7 +56,7 @@ Replace your `# Fabric Properties` section with the `gradle.properties` values f
 
 Next, you need to set your mod information. 
 
-Set `maven_group` to `com.github.yourusername` and set `archives_base_name` to the name of your mod, in lower-case letters and without spaces
+Set `maven_group` to `io.github.yourusername` and set `archives_base_name` to the name of your mod, in lower-case letters and without spaces
```
Comment (Tom-The-Geek) [inline docs/setup.md]:
```suggestion
Set `maven_group` to `io.github.yourusername` and set `archives_base_name` to the name of your mod, in lower-case letters and without spaces.
```

File: docs/libraries.md [hunk 4]
```diff
@@ -20,7 +20,7 @@ repositories {
 }
 ```
 
-You now add a maven repository with the URL you found to it:
+You now add a maven repository with the URL that links to it:
```
Comment (Tom-The-Geek) [inline docs/libraries.md]:
```suggestion
You now need to add the maven repository's URL to the section:
```

**Review by Tom-The-Geek — CHANGES_REQUESTED** [review]

File: docs/libraries.md (hunk 4 above)
Comment (NoComment1105) [reply docs/libraries.md]: Yeah I was most unsure on how to word that because it made little sense to me in the first place :P

**Comment by NoComment1105** [comment]
Changes made, open for review again

File: docs/mixin-injects.md [hunk 5]
```diff
@@ -22,13 +22,13 @@ This tells the compiler that this is a mixin class, targeting the class `net.min
...
+`@Inject` is a Mixin annotation. It tells the Mixin runtime to "inject" the logic of our method into the `onBreak` method we specified with the `method` argument. The `at = @At("HEAD")` part means that our code will be injected at the top of the method; making it the first thing to run run when that method is called.
 
 We then have a method with the same arguments as our target method, plus a `CallbackInfo` object.
 
 #### Registering Mixins
 
-When you add a mixin, you must put the mixin class in `<your_package>.mixin`. Then, you should add an entry to the `server` array in your `<your_modid>.mixins.json` file with the name of your mixin class.
+When you add a mixin, you must put the mixin class in `<your_package>.mixin`. Then, you should add an entry to the `server` array in your `<your_modid>.mixins.json` file with the name of your mixin class. Any mixins listed in the `server` array will only be loaded on a dedicated server
```
Comment (Tom-The-Geek) [inline docs/mixin-injects.md]:
```suggestion
When you add a mixin, you must put the mixin class in `<your_package>.mixin`. Then, you should add an entry to the `server` array in your `<your_modid>.mixins.json` file with the name of your mixin class. Any mixins listed in the `server` array will only be loaded on a dedicated server.
```

**Review by Tom-The-Geek — CHANGES_REQUESTED** [review]
Just a nitpick over a missing full stop, but it all looks good now :+1:

**Comment by NoComment1105** [comment]
> Just a nitpick over a missing full stop, but it all looks good now 👍

I swear I did that but sure :p
Thanks for the reviews

**Review by Tom-The-Geek — APPROVED** [review]

**Pull request merged**